api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/AnilistMediaRepository.kt

```diff
@@ -2,12 +2,11 @@ package com.imashnake.animite.api.anilist
 
 import com.apollographql.apollo3.ApolloClient
 import com.apollographql.apollo3.api.Optional
-import com.apollographql.apollo3.cache.normalized.FetchPolicy
-import com.apollographql.apollo3.cache.normalized.fetchPolicy
 import com.imashnake.animite.api.anilist.sanitize.media.Media
 import com.imashnake.animite.api.anilist.type.MediaSeason
 import com.imashnake.animite.api.anilist.type.MediaSort
 import com.imashnake.animite.api.anilist.type.MediaType
+import com.imashnake.animite.api.ext.asResult
```

Comment:
Is this import actually used? I don't see code added

Reply:
https://github.com/imashnake0/Animite/blob/e8c9cab7f446783c317598e444a601f560745470/api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/AnilistMediaRepository.kt#L47

Reply:
I moved it to a different package, hence the import.